fastapi/routing.py

```diff
@@ -178,6 +191,23 @@ def __init__(
 
 
 class APIRoute(routing.Route):
+    body_field: Optional[Field]
```

Comment:
I've added these as class annotations so that PyCharm doesn't complain about missing fields, since these fields are now defined in the setup functions, rather than the `__init__`. I would be happy to go with a different convention though (e.g. setting defaults in the `__init__` prior to calling the setup functions).